File: 5576_콘테스트.cpp
#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;
vector<int> wUniv;
vector<int> kUniv;

bool comp(int a, int b) {
    return a > b;
}

int main(void) {

    for(int i = 0; i < 10; i++) {
        int score;
        scanf("%d", &score);
        wUniv.push_back(score);
    }

    for(int i = 0; i < 10; i++) {
        int score;
        scanf("%d", &score);
        kUniv.push_back(score);
    }

    sort(wUniv.begin(), wUniv.end(), comp);
    sort(kUniv.begin(), kUniv.end(), comp);

    int wUnivScore = 0, kUnivScore = 0;
    for(int i = 0; i < 3; i++) {
        wUnivScore += wUniv[i];
    }

    for(int i = 0; i < 3; i++) {
        kUnivScore += kUniv[i];
    }

    printf("%d %d\n", wUnivScore, kUnivScore);

    return 0;
}

File: 5576_콘테스트.java
import java.io.BufferedReader;
import java.io.BufferedWriter;
import java.io.IOException;
import java.io.InputStreamReader;
import java.io.OutputStreamWriter;
import java.util.ArrayList;
import java.util.Arrays;
import java.util.Collections;

public class Main_5576 {

    public static void main(String...args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        Integer [] wUniv = new Integer[10];
        Integer [] kUniv = new Integer[10];

        for(int i = 0; i < 10; i++) {
            wUniv[i] = Integer.parseInt(br.readLine());
        }

        for(int i = 0 ; i < 10; i++) {
            kUniv[i] = Integer.parseInt(br.readLine());
        }

        int wUnivScore = 0, kUnivScore = 0;

        Collections.sort(Arrays.asList(wUniv), Collections.reverseOrder());
        Collections.sort(Arrays.asList(kUniv), Collections.reverseOrder());

        for(int i = 0 ; i < 3; i++) {
            wUnivScore += wUniv[i];
            kUnivScore += kUniv[i];
        }
        
        bw.write(wUnivScore + " " + kUnivScore + "\n");
        bw.flush();
        bw.close();
        br.close();
    }
    
}
